test/gui/tst_vfs/test.feature

```diff
@@ -1,3 +1,4 @@
+@skip
```

Comment:
there is an issue to unskip this: https://github.com/owncloud/client/issues/10072